redux/types/types.TransactionStatus.ts

```diff
@@ -0,0 +1,17 @@
+export enum PaymentStatus {
```

Comment:
Should add string values to the enums like: `Pending = "Pending"`

I believe the default to `1,2,3,...`.